Makefile

```diff
@@ -13,4 +16,33 @@ install-go-tools:
 
 .PHONY: format-code
 format-code: install-go-tools
-	goimports -w .
\ No newline at end of file
+	goimports -w .
+
+#----------------------------------------------------------------------------------
+# Tests
+#----------------------------------------------------------------------------------
+
+GINKGO_VERSION ?= 2.5.0 # match our go.mod
+GINKGO_ENV ?= GOLANG_PROTOBUF_REGISTRATION_CONFLICT=ignore ACK_GINKGO_DEPRECATIONS=$(GINKGO_VERSION)
+GINKGO_FLAGS := -v -tags=purego -compilers=4 -fail-fast -randomize-suites -randomize-all -skip-package=./installutils/kubeinstall,./debugutils/test
+GINKGO_REPORT_FLAGS := --json-report=test-report.json --junit-report=junit.xml
```

Comment:
Would it be worthwhile to document in a README or some such that these reports are generated (and how)?
It may be opaque to developers especially now that it's fully hidden in a Makefile and not referenced in code

Reply:
Yeah that's fair. What do you think is the best way to approach this? This will only be generated if you run `make test-with-coverage`. One option would be to document that target, and make it visible in `make help`. But another would be to create a README. I'm happy to for either, what do you think?

Reply:
Ah, I missed that part
Documenting `test-with-coverage` would be great, either in `make help` or README
I'd probably prefer README since it's the first place I'd look, but not a strongly held opinion